auditorium/views/main.js

```diff
@@ -60,16 +67,61 @@ function view (state, emit) {
     if (range.query || Object.keys(foreign).length) {
       url += '?' + new window.URLSearchParams(Object.assign(foreign, range.query))
     }
-    var anchor = html`<a href="${url}">${range.display}</a>`
+    var anchor = html`<a href="${url}" class="f5 b link dim ph3 pv2 mr2 mb1 dib br1 dark-green bg-black-05">${range.display}</a>`
     return html`
-      <li>
-        ${active ? html`<strong>${anchor}</strong>` : anchor}
+      <li class="mb1">
+        ${active ? html`<a href="${url}" class="f5 b link bb bw2 ph3 pv2 mr2 mb1 dib br1  dark-green bg-black-05">${range.display}</a>` : anchor}
       </li>
     `
   })
   var rangeSelector = html`
-    <h4>${__('Show data from the:')}</h4>
-    <ul>${ranges}</ul>
+      <h4 class ="f5 normal mt0 mb3">${__('Show data from the last')}</h4>
+      <ul class="flex flex-wrap list pl0 mt0 mb3">${ranges}</ul>
+    `
+
+  var goAccounts = html`
```

Comment:
Could we just override "manage" (maybe after renaming) in case `isOperator` is true instead of creating a variable that is never used when the view is rendered for a user?